### PR TITLE
Indicate toggle with icons, not only colors

### PR DESCRIPTION
**Pull request by henriettemoe**
To comply better with [WCAG Success Criteria 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)

### CODE REVIEW

**Comment by StianHaugland1** [comment]
Nice! Should we use "em" instead of "px"?

**Comment by henriettemoe** [comment]
> Nice! Should we use "em" instead of "px"?

Good point! Forgot about it since I used the react-toggle's css. But it is good to make it scale together with the font-size, like in Firefox where you have the option to size up the font-size.

File: h5p-bildetema/src/components/Toggle/ReactToggle.scss [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  cursor: pointer;
+  background-color: transparent;
+  border: 0;
+  padding: 0;
+
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```
Comment (boyum) [inline h5p-bildetema/src/components/Toggle/ReactToggle.scss:17]: Do we need all these, or are they added automatically?

File: h5p-bildetema/src/components/Toggle/ReactToggle.scss [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+.react-toggle-track {
+  width: 3.125em;
+  height: 1.5em;
+  padding: 0;
+  border-radius: 1.875em;
+  background-color: #4d4d4d;
+  -webkit-transition: all 0.2s ease;
+  -moz-transition: all 0.2s ease;
```
Comment (boyum) [inline h5p-bildetema/src/components/Toggle/ReactToggle.scss:49]:
```suggestion
```

File: h5p-bildetema/src/components/Toggle/ReactToggle.scss [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  height: 0.625em;
+  top: 0;
+  bottom: 0;
+  margin-top: auto;
+  margin-bottom: auto;
+  line-height: 0;
+  left: 0.5em;
+  opacity: 0;
+  -webkit-transition: opacity 0.25s ease;
+  -moz-transition: opacity 0.25s ease;
```
Comment (boyum) [inline h5p-bildetema/src/components/Toggle/ReactToggle.scss:77]:
```suggestion
```

File: h5p-bildetema/src/components/Toggle/ReactToggle.scss [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  svg {
+    height: 0.625em;
+    width: 0.875em;
+  }
+}
+
+.react-toggle--checked .react-toggle-track-check {
+  opacity: 1;
+  -webkit-transition: opacity 0.25s ease;
+  -moz-transition: opacity 0.25s ease;
```
Comment (boyum) [inline h5p-bildetema/src/components/Toggle/ReactToggle.scss:89]:
```suggestion
```

File: h5p-bildetema/src/components/Toggle/ReactToggle.scss [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+  height: 0.625em;
+  top: 0;
+  bottom: 0;
+  margin-top: auto;
+  margin-bottom: auto;
+  line-height: 0;
+  right: 0.625em;
+  opacity: 1;
+  -webkit-transition: opacity 0.25s ease;
+  -moz-transition: opacity 0.25s ease;
```
Comment (boyum) [inline h5p-bildetema/src/components/Toggle/ReactToggle.scss:105]:
```suggestion
```

File: h5p-bildetema/src/components/Toggle/ReactToggle.scss [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+  border-radius: 50%;
+  background-color: #ffffff;
+
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+
+  -webkit-transition: all 0.25s ease;
+  -moz-transition: all 0.25s ease;
+  transition: all 0.25s ease;
```
Comment (boyum) [inline h5p-bildetema/src/components/Toggle/ReactToggle.scss:135]:
```suggestion
  background-color: #ffffff;
  box-sizing: border-box;
  transition: all 0.25s ease;
```

File: h5p-bildetema/src/components/Toggle/ReactToggle.scss (hunk 1 above)
Comment (henriettemoe) [reply h5p-bildetema/src/components/Toggle/ReactToggle.scss:17]: They are from the original css for the react-toggle, but we can remove them

File: h5p-bildetema/src/components/Toggle/ReactToggle.scss [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+.react-toggle {
+  touch-action: pan-x;
+
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+  background-color: transparent;
+  border: 0;
+  padding: 0;
+  user-select: none;
```
Comment (boyum) [inline h5p-bildetema/src/components/Toggle/ReactToggle.scss:12]: I think we could keep the `-webkit-touch-callout: none;`
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-touch-callout

```suggestion
  user-select: none;
  -webkit-touch-callout: none;
```

**Review by boyum — APPROVED** [review]

**Pull request merged**